packages/peregrine/lib/talons/ProductImageCarousel/useThumbnail.js

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from 'react';
+
+export const useThumbnail = props => {
+    const { itemIndex, onClickHandler } = props;
```

Comment:
I think this kind of talon is going to prove to be a good candidate for a utility hook in the future. 👍